### ISSUE TITLE
`Label(_:systemImage:)` is not supported

### ISSUE THREAD

**Issue by ikesyo** [nalexn/ViewInspector#176]
```swift
let view = Label("tx", systemImage: "eye")
let text = try view.inspect().label().icon().image().labelView()
```

This throws an error:

> caught error: "label().icon().image().labelView() found Image.ImageLabel instead of Text"

**Comment by nalexn** [comment]
Thanks for the repport! This is fixed in the coming new version

**Comment by nalexn** [comment]
Released with v0.9.2
